drgn_tools/util.py

```diff
@@ -113,6 +113,21 @@ def type_exists(prog: Program, type_: str) -> bool:
         return False
 
 
+def variable_exists(prog: Program, variable: str) -> bool:
+    """
+    Check whether global variable exists in drgn
+
+    :param prog: drgn program
+    :param variable: variable name
+    :returns: true if exist, otherwise false
+    """
+    try:
+        prog.variable(variable)
+        return True
+    except LookupError:
+        return False
+
+
```

Comment:
Looking at the drgn-tools code, the convention for this seems to be `if "varname" in prog:`. Your function is more technically correct (as `"varname" in prog` would also check for functions and constants). But I think it may be more consistent to stick with the way the rest of the code does it.